todo2.js

```diff
@@ -0,0 +1,82 @@
+var tasksArray = [];
+
+document.addEventListener("DOMContentLoaded", function () {
+  let taskInput = document.getElementById("input");
```

Comment:
Move the element to the upper scope. So you won't have to pass them to all the functions.